packages/cli/src/lib/snapshot/snapshot.spec.ts

```diff
@@ -0,0 +1,83 @@
+jest.mock('../platform/authenticatedClient');
+jest.mock('fs');
+
+import {createReadStream, ReadStream} from 'fs';
+import {join} from 'path';
+import {Readable} from 'stream';
+import {mocked} from 'ts-jest/utils';
+import {AuthenticatedClient} from '../platform/authenticatedClient';
+import {SnapshotFactory} from './snapshotFactory';
+
+const mockedCreateReadStream = mocked(createReadStream);
+const mockedAuthenticatedClient = mocked(AuthenticatedClient);
+const mockedCreateSnapshotFromFile = jest.fn();
+
+const doMockReadStream = () => {
+  mockedCreateReadStream.mockImplementation(() => {
+    const buffer = Buffer.from('this is a tést');
+    const readable = new Readable();
+    readable._read = () => {};
+    readable.push(buffer);
+    readable.push(null);
+    return readable as unknown as ReadStream;
+  });
+};
```

Comment:
```suggestion
const doMockReadStream = () => {
  mockedCreateReadStream.mockImplementation(() =>
    jest
      .requireActual<{createReadStream: typeof createReadStream}>('fs')
      .createReadStream(Buffer.from('this is a tést'))
  );
};
```

You can also remove `ReadStream` and `Readable` from your imports.

Reply:
Wait a second, this will fail because no such file or directory named 'this is a tést' exist.
```

Error: ENOENT: no such file or directory, open 'this is a tést'
Emitted 'error' event on ReadStream instance at:
    at internal/fs/streams.js:147:14
    at FSReqCallback.oncomplete (fs.js:171:23) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: 'this is a tést'
}
events.js:292
```

Reply:
Hm true, I thought i would 'streamify' the buffer